### PR TITLE
CIV-13182 View mediation settlement agreement 

### PR DESCRIPTION
**Pull request by LeonardoPalmeiro**
### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-13185
https://tools.hmcts.net/jira/browse/CIV-13182

### Change description ###
Claimant 

localhost_3001_dashboard_1713168471123613_claimantNewDesign

Defendant
localhost_3001_dashboard_1713168471123613_defendant

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/dashboard/dashboardInterpolationService.ts [hunk 1]
```diff
@@ -88,6 +88,7 @@ const setDashboardValues = (claim: Claim, claimId: string, notificationId?: stri
   valuesMap.set('{HEARING_DUE_DATE}', claim?.caseProgressionHearing?.hearingDate? claim.bundleStitchingDeadline: '');
   valuesMap.set('{APPLY_HELP_WITH_FEES_START}', APPLY_HELP_WITH_FEES_START.replace(':id', claimId));
   valuesMap.set('{VIEW_CCJ_REPAYMENT_PLAN_CLAIMANT}', CCJ_REPAYMENT_PLAN_CLAIMANT_URL.replace(':id', claimId));
+  valuesMap.set('{VIEW_MEDIATION_SETTLEMENT_AGREEMENT}', VIEW_MEDIATION_SETTLEMENT_AGREEMENT_DOCUMENT.replace(':id', claimId));
```
Comment (laarmada) [inline src/main/services/dashboard/dashboardInterpolationService.ts:91]: is the same for claimant and defendant?

**Review by laarmada — APPROVED** [review]
minor comment

File: src/main/services/dashboard/dashboardInterpolationService.ts (hunk 1 above)
Comment (LeonardoPalmeiro) [reply src/main/services/dashboard/dashboardInterpolationService.ts:91]: the URL is the same there is no distinction between the parties.
When we upload mediation settlement agreement the document should be available to both parties.

Comment (laarmada) [reply src/main/services/dashboard/dashboardInterpolationService.ts:91]: Cool, thanks Leo

**Review by sankaviv1 — DISMISSED** [review]

**Review by sankaviv1 — DISMISSED** [review]

**Review by LTurkingtonHMCTS — DISMISSED** [review]

**Review by laarmada — APPROVED** [review]

**Pull request merged**